app/scripts/react-directives/common.jsx

```
@@ -130,19 +137,23 @@ export const Checkbox = ({ label, value, onChange, disabled }) => {
-      ref={ref}
-      className="form-control"
-      type="text"
-      placeholder={placeholder}
-      value={value}
-      onChange={onChange}
-      disabled={disabled}
-    />
-  );
-});
+export const LineEdit = React.forwardRef(
+  ({ placeholder, value, onChange, disabled, type, name, required }, ref) => {
```

ESLint

[error] Expected a line break after this opening brace.

(object-curly-newline)